packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -1,33 +1,49 @@
 <template>
-    <div class="certifaction-verification"
-         @dragover.prevent="dragOver"
-         @dragleave="dragLeave"
-         @drop.prevent="handleDrop">
+    <div class="verification-container">
+        <div v-if="digitalTwin" class="certifaction-digital-twin">
+            <input id="fileItem" type="file" style="position: absolute; overflow: hidden; z-index: 99;">
+            <!-- JUST TEMP -->
+            <DigitalTwin
+                :filtered-verification-items="filteredVerificationItems"
+                :file-name="digitalTwinFileName"
+                :verifier-information="verifierInformation"
+                :certifaction-api-url="certifactionApiUrl"
+                @process-file="processDigitalTwinFile">
+            </DigitalTwin>
```

Comment:
For me, this should be inside certifaction-verification and not separate. The drag events need to be adjusted of course when in the digital twin mode.